.darglint

```diff
@@ -0,0 +1,4 @@
+[darglint]
+docstring_style=sphinx
+enable=DAR104
```

Comment:
@cidrblock remembering that this project uses type annotations, I'm leaning toward dropping this. WDYT?
```suggestion
```

Reply:
yep makes sense

"DAR104: (disabled) The docstring parameter has no type specified"

I kind of wish it required that types in the docstring weren't present at all :)

Reply:
@cidrblock we could make a feature request and/or think of having our own linter plugin (it shouldn't be too hard to make one).